brainscore_language/plugin_management/conda_score.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pickle
+import tempfile
+
+from brainscore_core.metrics import Score
+from brainscore_language.plugin_management.environment_manager import EnvironmentManager
+
+TMP_DIR = tempfile.TemporaryDirectory()
+SCORE_PATH = f'{TMP_DIR.name}_score.pkl'
+
+class CondaScore(EnvironmentManager):
+    """ run scoring in conda environment """
+    def __init__(self, plugin_ids):
+        super(CondaScore, self).__init__()
+
+        self.model = plugin_ids['model']
+        self.benchmark = plugin_ids['benchmark']
+        self.env_name = f'{self.model}_{self.benchmark}'
+        self.script_path = f'{Path(__file__).parent}/conda_score.sh'
+        self.result = self.score_in_env()
+
+    def score_in_env(self) -> int:
```

Comment:
I don't think the returned process is an int; you're not returning the code, but rather the process

Reply:
You're correct, this is outdated (was previously returning `returncode`). Done.